builds/jazz-build-module/service-configuration-data-loader.groovy

```diff
@@ -440,77 +492,73 @@ def loadServiceConfigurationData() {
         }
 
         if (service_name.trim() == "jazz_admin") {
+            sh "sed -i -- 's/{scm-type}/${config_loader.SCM.TYPE}/g' ./config/dev-config.json"
+            sh "sed -i -- 's/{scm-type}/${config_loader.SCM.TYPE}/g' ./config/stg-config.json"
+            sh "sed -i -- 's/{scm-type}/${config_loader.SCM.TYPE}/g' ./config/prod-config.json"
+
             sh "sed -i -- 's/{jazz_admin}/${config_loader.JAZZ.ADMIN}/g' ./config/dev-config.json"
             sh "sed -i -- 's/{jazz_admin}/${config_loader.JAZZ.ADMIN}/g' ./config/stg-config.json"
             sh "sed -i -- 's/{jazz_admin}/${config_loader.JAZZ.ADMIN}/g' ./config/prod-config.json"
-        }
 
-        if (service_name.trim() == "jazz_email") {
-            sh "sed -i -- 's/{conf-region}/${region}/g' ./config/dev-config.json"
-            sh "sed -i -- 's/{conf-region}/${region}/g' ./config/stg-config.json"
-            sh "sed -i -- 's/{conf-region}/${region}/g' ./config/prod-config.json"
-        }
+            if (config_loader.SCM.TYPE == "bitbucket") {
+                sh "sed -i -- 's,{base-url},http://${config_loader.REPOSITORY.BASE_URL},g' ./config/dev-config.json"
+                sh "sed -i -- 's,{base-url},http://${config_loader.REPOSITORY.BASE_URL},g' ./config/stg-config.json"
+                sh "sed -i -- 's,{base-url},http://${config_loader.REPOSITORY.BASE_URL},g' ./config/prod-config.json"
+
+                withCredentials([[$class: 'UsernamePasswordMultiBinding', credentialsId: config_loader.REPOSITORY.CREDENTIAL_ID, passwordVariable: 'PWD', usernameVariable: 'UNAME']]) {
+                    sh "sed -i -- 's/{bb_username}/${UNAME}/g' ./config/dev-config.json"
+                    sh "sed -i -- 's/{bb_username}/${UNAME}/g' ./config/stg-config.json"
+                    sh "sed -i -- 's/{bb_username}/${UNAME}/g' ./config/prod-config.json"
+
+                    sh "sed -i -- 's/{bb_password}/${PWD}/g' ./config/dev-config.json"
+                    sh "sed -i -- 's/{bb_password}/${PWD}/g' ./config/stg-config.json"
+                    sh "sed -i -- 's/{bb_password}/${PWD}/g' ./config/prod-config.json"
+                }
+            }
 
-        if ((config_loader.SLACK.ENABLE_SLACK == "true") && (service_name.trim() == "jazz_slack-channel")) {
```

Comment:
We'll need these changes.

Overall, I believe something went wrong during the merge of this file here. Please take a look at the diff again and add code segments that are removed during the branch sync.

Reply:
these lines were removed as part of #535. I will be adding them back in

Reply:
fixed